<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFoldingClass</class>
 <widget class="QMainWindow" name="CFoldingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFolding</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="MoveTool">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="VertexTool">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="EdgeTool">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>371</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="show2dLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>431</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="show3dLayout"/>
   </widget>
   <widget class="QScrollArea" name="candiLayout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>851</width>
      <height>141</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>849</width>
       <height>122</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="SymmetryTool">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="mergeTool">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>O</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="faceTool">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_2"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionInitialize"/>
   <addaction name="actionOptimize"/>
   <addaction name="actionLoadtest"/>
  </widget>
  <action name="actionInitialize">
   <property name="text">
    <string>Initialize</string>
   </property>
  </action>
  <action name="actionLoad_2">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOptimize">
   <property name="text">
    <string>Optimize</string>
   </property>
  </action>
  <action name="actionLoadtest">
   <property name="text">
    <string>loadtest</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CFolding.qrc"/>
 </resources>
 <connections/>
</ui>
